it/uk/gov/hmrc/fileupload/support/FileActions.scala

```diff
@@ -66,10 +63,10 @@ trait FileActions extends ActionsSupport {
         "X-Requested-With" -> "someId")
       .post(s"-----011000010111000001101001\r\n" +
         "Content-Disposition: form-data; name=\"file1\"; filename=\"test.pdf\"\r\n" +
-        "Content-Type: application/pdf\r\n\r\n" +
+        "Content-Type: application/pdf\r\n" +
```

Comment:
why we had/have \r\n ? It look like windows new line.

Reply:
"\r\n" was there for sometimes now, i believe that is the way to create a text file, as without "\r\n" file will not created